CMakeLists.txt

```diff
@@ -148,7 +148,7 @@ option(SWIFT_INCLUDE_DOCS
     TRUE)
 
 set(_swift_include_apinotes_default FALSE)
-if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+if(CMAKE_SYSTEM_NAME STREQUAL "Darwin" OR CMAKE_SYSTEM_NAME STREQUAL "Windows")
   set(_swift_include_apinotes_default TRUE)
 endif()
 
```

Comment:
The description of the option here is misleading, it isn't about inclusion in the toolchain it seems (based on apple/swift-installer-scripts#79) but rather into the installed image.  Could you please update the comment?

Reply:
`apinotes` are regarded to be optional overlays, so we can choose to include or not include it in our build.

Reply:
Sure; but that doesn't change the fact that the comment for the CMake GUI is misleading.